app/services/auto-config/brand-device.ts

```diff
@@ -144,56 +148,6 @@ export class BrandDeviceService extends StatefulService<IBrandDeviceState> {
     }
   }
 
-  /**
-   * This method is calling by autoconfiguration commands from onboarding_cmds.json
-   * @Example
-   * addSceneItem(
-   *  'camera',
-   *  'dshow_input',
-   *  {
-   *   sourceSetting: {use_custom_audio_device: true}
-   *   fuzzySourceSettings: { audio_device_id: 'Microphone (Realtek(R) Audio)' }} // use description instead of id here
-   * )
-   */
-  addSceneItem(
-    name: string,
-    type: TSourceType,
-    options: {
-      sourceSettings?: Dictionary<any>,
-      sourceFuzzySettings?: Dictionary<any>,
-      audioSettings: Partial<IAudioSource>
-    })
-  {
-    const sceneItem = this.scenesService.activeScene.createAndAddSource(name, type);
-    const source = sceneItem.getSource();
-
-    if (!options.sourceSettings) return;
-    const settings = cloneDeep(options.sourceSettings);
-    const propsFormData = source.getPropertiesFormData();
-
-    for (const prop of propsFormData) {
-      // handle only LIST props for now
-      if (prop.type !== 'OBS_PROPERTY_LIST') continue;
-      if (!(prop.name in options.sourceFuzzySettings)) continue;
-
-      const searchPattern = options.sourceFuzzySettings[prop.name];
-
-      const option = (prop as IObsListInput<string>).options.find(option => {
-        return (option.value.includes(searchPattern) || option.description.includes(searchPattern))
-      });
-
-      if (!option) continue;
-      settings[prop.name] = option.value;
-    }
-
-    source.updateSettings(settings);
-
-    if (!options.audioSettings) return;
-    const audioSource = this.audioService.getSource(source.sourceId);
-    if (!audioSource) return;
-    audioSource.setSettings(options.audioSettings);
-  }
```

Comment:
^ this method has been removed since `SourcesService` can do all these things now